common/config/rush/version-policies.json

```diff
@@ -0,0 +1,80 @@
+/**
+ * This is configuration file is used for advanced publishing configurations with Rush.
+ * For full documentation, please see https://rushjs.io
+ */
+[
+  /**
+   * A "version policy" is a custom package versioning strategy for use with "rush publish".
+   * The strategy affects a set of projects that are specified using the "versionPolicyName" field
+   * in rush.json.
+   */
+  {
+    /**
+     * (Required) Indicates the kind of version policy being defined ("lockStepVersion" or "individualVersion").
+     *
+     * The "lockStepVersion" mode specifies that the projects will use "lock-step versioning".  This
+     * strategy is appropriate for a set of packages that act as selectable components of a
+     * unified product.  The entire set of packages are always published together, and always share
+     * the same NPM version number.  When the pacakges depend on other packages in the set, the
+     * SemVer range is usually restricted to a single version.
+     */
+    "definitionName": "lockStepVersion",
+
+    /**
+     * (Required) The name that will be used for the "versionPolicyName" field in rush.json.
+     * This name is also used command-line parameters such as "--version-policy"
+     * and "--to-version-policy".
+     */
+    "policyName": "MyBigFramework",
+
+    /**
+     * (Required) The current version.  All packages belonging to the set should have this version
+     * in the current branch.  When bumping versions, Rush uses this to determine the next version.
+     * (The "version" field in package.json is NOT considered.)
+     */
+    "version": "1.0.0",
+
+    /**
+     * (Required) The type of bump that will be performed when publishing the next release.
+     * When creating a release branch in Git, this field should be updated according to the
+     * type of release.
+     *
+     * Valid values are: "prerelease", "release", "minor", "patch", "major"
+     */
+    "nextBump": "prerelease",
```

Comment:
>prerelease [](start = 17, length = 10)

who chooses the prerelease name? what is the "release" value for?

Reply:
This topic is so complex that I'd rather document it on the web site.

---
In reply to: [217226585](https://github.com/Microsoft/rush-example/pull/2#discussion_r217226585) [](ancestors = 217226585)